core/libraries/batch/JobHandlers/RegistrationsReport.php

```diff
@@ -47,6 +47,7 @@ class RegistrationsReport extends JobHandlerFile
     public function create_job(JobParameters $job_parameters)
     {
         $event_id = intval($job_parameters->request_datum('EVT_ID', '0'));
+        $DTT_id = intval($job_parameters->request_datum('DTT_ID', '0'));
```

Comment:
he he.. Michael loved to use `intval()`  but it's a function from PHP 4 and it's advised to not use it since it may get deprecated at some point. It's better to simply type hint values. 

```php
        $event_id = (int) $job_parameters->request_datum('EVT_ID', '0');
        $DTT_id   = (int) $job_parameters->request_datum('DTT_ID', '0');
```

Reply:
Nice!